src/store.js

```diff
@@ -91,7 +91,20 @@ export default new Vuex.Store({
 		initValue(state, payload) {
 			//console.log("store init for", payload.p, "payload:", payload, "state:", state)
```

Comment:
remove this line at the same time, there is another commented out line below as well. did miss this one previously